ast/misc.go

```diff
@@ -626,6 +626,26 @@ type HandleRange struct {
 	End   int64
 }
 
+// ShowLogType defines the type for SlowLog.
+type ShowLogType int
```

Comment:
s/ShowLogType/ShowSlowQueryType/

Reply:
When I merge master, this would be solved.